README.md

```diff
@@ -127,6 +127,109 @@ The following options are available:
 : A SQL statement that is executed before inserting the data, e.g. to create the target table.
 : *NOTE:* the underlying database should support the *create table if not exist* idiom, as the plugin will execute this statement every time the script is run.
 
+## SQL Execution Functions
+
+This plugin provides the following functions for executing SQL statements that don't return data, such as DDL (Data Definition Language) and DML (Data Manipulation Language) operations.
+
+### execute
+
+The `execute` function executes a SQL statement that doesn't return a result set, such as `CREATE`, `ALTER`, `DROP`, `INSERT`, `UPDATE`, or `DELETE` statements. For example:
+
+```nextflow
+include { execute } from 'plugin/nf-sqldb'
+
+// Create a table
+execute(
+    db: 'foo',
+    statement: '''
+        CREATE TABLE IF NOT EXISTS sample_table (
+            id INTEGER PRIMARY KEY,
+            name VARCHAR(100),
+            value DOUBLE
+        )
+    '''
+)
+
+// Insert data
+execute(
+    db: 'foo',
+    statement: "INSERT INTO sample_table (id, name, value) VALUES (1, 'alpha', 10.5)"
+)
+
+// Delete data
+execute(
+    db: 'foo',
+    statement: "DELETE FROM sample_table WHERE id = 1"
+)
+```
+
+The following options are available:
+
+`db`
+: The database handle. It must be defined under `sql.db` in the Nextflow configuration.
+
+`statement`
+: The SQL statement to execute. This can be any DDL or DML statement that doesn't return a result set.
+
+### executeUpdate
+
+The `executeUpdate` function is similar to `execute`, but it returns the number of rows affected by the SQL statement. This is particularly useful for DML operations like `INSERT`, `UPDATE`, and `DELETE` where you need to know how many rows were affected. For example:
```

Comment:
Also here `sqlExecuteUpdate` can be better. 

I also wonder if it's really necessary having a separate function only to return the number of modified rows? it may be enough having just one function and return `null` for DDL statement, or even `1` to signal a success. Who is invoking it should know how to to interpret the result value.